### PR TITLE
Screenshots

### PR DESCRIPTION
**Pull request by ivy-lli**
- Enable some screenshot tests
- Use pageobjects for screenshot tests
- Remove some utils
- Screenshot only accordion instead of full page

### CODE REVIEW

File: integrations/standalone/tests/pageobjects/Composite.ts [hunk 1]
```diff
@@ -60,4 +60,8 @@ export abstract class Composite {
   tags(): any {
     return new Tags(this.page, this.locator);
   }
+
+  currentLocator() {
```
Comment (weissreto) [inline integrations/standalone/tests/pageobjects/Composite.ts:64]: locator() does not work because locator is already a property?

Comment (weissreto) [reply integrations/standalone/tests/pageobjects/Composite.ts:64]: then it is maybe better to rename the property and make it private. And use locator() outside of the class

**Review by weissreto — APPROVED** [review]
Looks good. 2 small comments

File: integrations/standalone/tests/pageobjects/Composite.ts (hunk 1 above)
Comment (ivy-lli) [reply integrations/standalone/tests/pageobjects/Composite.ts:64]: Yes, and yes. I will change that but I think in a further PR as I have many open PRs which may result in merge conflicts 🙈

**Pull request merged**